makefiles/boards/alias.inc.mk

```diff
@@ -0,0 +1,10 @@
+# list of known alias
+ALIAS.phynode   = pba-d-01-kw2x
+#TODO remove this line (as its just for show)
+ALIAS.linux     = native
+
+# this temporary variable is needed as we can not use BOARD when assigning BOARD
+ALIAS_TARGET := $(ALIAS.$(BOARD))
+ifneq (,$(ALIAS_TARGET))
+  override BOARD = $(ALIAS_TARGET)
```

Comment:
Doesn't this have to be ```BOARD := $(ALIAS_TARGET)```?

Reply:
In general yes, and also tried that, but does not solve the problem above.